### ISSUE TITLE
Wrong resolution for waveshare_epd.epd7in3g, 168x400 but should be 800x480

### ISSUE THREAD

**Issue by reinharty** [robweber/omni-epd#99]
Hello @robweber ,
thank you for this library.
I believe I might have found a bug in the implementation of the epd7in3g. Its a B/W,Red,Yellow screen.
When running the basic and init examples, it only draws to a small area instead of fullscreen:
![PXL_20240622_133503155 jpg_compressed](https://github.com/robweber/omni-epd/assets/15419390/693c0c7d-7750-4e0d-80c5-2fdc39ce4320)

Using the official Waveshare_epd example for the same screen, it draws to the whole screen:
![PXL_20240622_140540571 jpg_compressed](https://github.com/robweber/omni-epd/assets/15419390/8f33e447-750f-4643-a839-b2cc917a6df0)
(Ironically I only managed to get this to work after installing your module.)

Printing width and height results in 168 and 400 but should be 800x480. 
Sadly I have been unable to identify where these values are loaded from.

**Comment by robweber** [comment]
Thanks for all of the information on this. I was able to locate the issue pretty quickly. Looks like the [wrong Waveshare lib](https://github.com/robweber/omni-epd/blob/main/src/omni_epd/displays/waveshare_display.py#L242C42-L242C50) is being loaded for this display. For any of the displays that aren't checked as "confirmed working" this is pretty common to find a small thing. We did the best we could trying to translate all the individual driver falls. 

I should be able to fix this up quick and get it in the repo. If you wouldn't mind confirming the fix after it's done I can mark this display as tested.
